src/applications/disability-benefits/all-claims/tests/pages/militaryDutyImpact.unit.spec.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { mount } from 'enzyme';
+
+import {
+  DefinitionTester,
+  selectRadio,
+} from '../../../../../platform/testing/unit/schemaform-utils';
+import formConfig from '../../config/form';
+import { ERR_MSG_CSS_CLASS } from '../../constants';
+
+describe('Recent Job Applications', () => {
+  const page = formConfig.chapters.disabilities.pages.militaryDutyImpact;
+  const { schema, uiSchema } = page;
+
+  it('should render', () => {
+    const form = mount(
```

Comment:
going to need to update the tests to add `form.unmount()` at the very end due to new linter issues.